src/videojs-contrib-hls.js

```diff
@@ -273,6 +276,12 @@ export default class HlsHandler extends Component {
 const HlsSourceHandler = function(mode) {
   return {
     canHandleSource(srcObj) {
+      // this will video.js to to not use this tech/mode if its
+      // not the one we have been overriden to use. It does this by saying that
+      // the tech you don't want to use can't handle this source
```

Comment:
This comment needs some love: "this will video.js to to not use" and "overriden".